components.json

```diff
@@ -757,6 +757,14 @@
 			"title": "R",
 			"owner": "Golmote"
 		},
+		"turtle": {
+			"title": "Turtle",
```

Comment:
With the new title, you'll also have to adjust the position. Before Twig, I think.

Reply:
Right, done.